rust/hyperlane-core/src/accumulator/incremental.rs

```diff
@@ -105,7 +105,7 @@ mod test {
             // insert the leaves
```

Comment:
Looks like this test case is failing if you run `cargo test`

Reply:
hm, that's because `test_utils::load_merkle_test_json()` fails. It's super weird bc that function didn't change since it was added, nor did `merkle.json` (which it reads from and just calls `serde_json::from_str` on). Can't tell what's wrong yet